six/modules/c++/six.sicd/source/SICDByteProvider.cpp

```diff
@@ -130,6 +130,28 @@ const void* SICDByteProvider::NITFBufferList::getBlock(
     return NULL;
 }
 
+size_t SICDByteProvider::getNumBytesPerRow(
+        const NITFWriteControl& writer) const
```

Comment:
This can be `static` rather than `const` right?

Reply:
Done.